glide.yaml

```diff
@@ -41,7 +41,7 @@ import:
 - package: github.com/uber-go/atomic
   version: e682c1008ac17bf26d2e4b5ad6cdd08520ed0b22
 - package: github.com/uber-go/tally
-  version: 3.0.0
+  version: ^3.0.0 # ie >= 3.0.0, < 4.0.0
```

Comment:
Hmm, I was going off this example:

```
^1.2.3 is equivalent to >= 1.2.3, < 2.0.0
```

I took that to mean 

```
^3.0.0 is equivalent to >= 3.0.0, < 4.0.0
```

Can certainly use the `^3.` since that's simpler (just want to understand it myself).